docs/Course Wiki/ECE Course Offerings/ECE476.md

```diff
@@ -2,41 +2,58 @@
 
+Next students are introduced to the idea of economic dispatch (tertiary control) and learn about cost function modeling of generators. Control and mathematical optimization concepts are introduced as students learn about the economics behind power generation within our modern day grid. 
 
+Unbalanced system operation is introduced to students through the method of symmetrical components. Students learn about the different types of faults that can occur within a three phase system (single line-ground, line-line, double line-ground, open, balanced three-phase-ground) and how to model them using linear algebra techniques as well as basic circuit analysis. 
 
+Finally, students go back to generator modeling and dive deep into transient stability during fault conditions. The governing differential equations for generators are studied in depth and rigorously analyzed to build up to the equal area criterion for transient stability analysis. Parallels are drawn between generator transients and mass-spring systems to help students conceptualize the difficult topics. 
+## Prerequisites
+-[ECE330](ECE330.md)
+This class heavily relies on content from that class. There are many topical prerequisites for this class as it relies on familiarity of differential equations, matrix algebra, and computer programming. 
+## When to Take It
+Take this course soon after [ECE330](ECE330.md).
 ## Course Structure
-
-
-
+This course has two midterms and a cumulative final. Homework is given weekly. 
 ## Instructors
-
-
+The course director for ECE476 is Alejandro Dominguez-Garcia. He is a professional in the field of power systems and has been teaching this class for many years. The content has changed over the years, but Professor Dominguez-Garcia has his own set of notes that he regularly updates. 
 
 ## Course Tips
-
+Read the notes before the lectures as Professor Dominguez-Garcia goes through content at a fast pace.
 
 
 ## Life After
-
+If you like the generator modeling and transient generator response section of this course, take [ECE431](ECE431.md). The direct next course is Large-Scale System Analysis (ECE 530) which is also only offered in the Fall.
 
 
 ## Infamous Topics
-
+Unbalanced system modeling is a complicated topic that requires much more time to understand than other topics within this course. 
```

Comment:
Make a bullet point for unbalanced system modeling.  Also do you have any tips for someone who struggles with it?